note/src/components/Notes.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
+
+const NotesContainer = styled.div`
+  border: 1px solid black;
+  margin-left: 20%;
+  height: 100%;
+`;
+
+ const Heading = styled.h2`
+  display: flex;
+  margin-left: 30px;
+`;
+
+ const NotesDisplay = styled.div`
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+`;
+
+ const NoteCard = styled.div`
+  border: 1px solid gray;
+  width: 225px;
+  height: 200px;
+  margin-left: 60px;
+  margin-top: 30px;
+`;
+
+ const NoteTitle = styled.h3`
+  border-bottom: 1px solid gray;
+  display: flex;
+  margin-left: 10px;
+  margin-right: 10px;
+  text-decoration: none;
+  color: black;
+`;
+
+ const NoteBody = styled.p`
+  display: flex;
+  margin-left: 10px;
+  margin-right: 10px;
+  text-decoration: none;
+  color: gray;
+`;
+
+const Notes = props =>  {
+    console.log(props.notes)
+        return (
+            <NotesContainer>
+                <Heading>Your Notes:</Heading>
+                <NotesDisplay>
+                    {props.notes.map(note => {
+                        return (
+                            <Link to={`/note/${note._id}`} style={{textDecoration: 'none'}}>
```

Comment:
you need your key to be in your Link, not in your NoteCard, since the key goes in the highest item in your map